### ISSUE TITLE
The bot died from asphyxia at a mining station after theft oxygen mask and was caught.

### ISSUE THREAD

**Issue by pavel4win** [FakeFishGames/Barotrauma#13424]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The situation occurred in the network game
At the mine station, a bot wandered into a mine itself.
When the gas went there the bot took the mask from the locker.
He was immediately caught by a security guard who took away his weapon.
Then the guard put handcuffs on the bot.
That's why the bot couldn't put on the mask.
Then bot died of asphyxia.
Was this how it was intended?
In my opinion a very bad idea.
It would be better if the bot (if do not have mask) simply left the mine if there is nothing to breathe there and did not steal things.

It also seems to me that the ability to put on and take off a mask (and any headgear) should be made available even to a person in handcuffs, because in reality it is possible to remove a mask or helmet from a belt and put it on the face even with handcuffs on hands

Last question is: why do the security take away the weapons but not the stolen items?

### Reproduction steps

At the mining station, take the bot to the mine well without a mask in his inventory and leave it there.
Wait until the irreparable happens.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

While I agree this is a bit strange, I think it's such a rare edge case that it's not worth the trouble to add extra complexity to the bot AI to work around it. Arguably, it makes sense that the bot tries to steal a mask when they notice they're running low on oxygen, and an overeager security officer might not like that and arrest the bot.

>It also seems to me that the ability to put on and take off a mask (and any headgear) should be made available even to a person in handcuffs, because in reality it is possible to remove a mask or helmet from a belt and put it on the face even with handcuffs on hands

I don't quite agree with this. The character's arms are behind their back when handcuffed, so I think it'd take pretty unusual amount of flexibility to put on your mask. And I don't think it would make sense to add special rules for which items can be equipped/unequipped while wearing handcuffs: it's much more simpler from the player's perspective that the same rules apply to all items.

>Last question is: why do the security take away the weapons but not the stolen items?

They should also take off the stolen items. I gave this a quick test and did not notice any issues with it.

**Comment by Regalis11** [comment]
I did a quick change to somewhat address this (commit in our private repo: https://github.com/FakeFishGames/Barotrauma-development/commit/7cba6d3f8822bafea8a6be825bab08e53ab50521). The bots now avoid staying in the non-ventilated hulls in the mine modules, so this should be much less likely to occur.

**Comment by pavel4win** [comment]
Thanks for the quick response and bug fix.

The situation is really rare, but I've already had it twice in 3,452 hours of play. That's why I wrote about the situation.
I have a save in which the bot is handcuffed and almost dead from asphyxia.
The bot still has the stolen mask and oxygen tank in its inventory.
So I was wondering:
"why do the security take away the weapons but not the stolen items?"
But in fact, I don't really care about this issue, since I never steal on stations.
Perhaps the guard's inventory was full at the time when I saving the game?
Fortunately, I managed to save the bot's life this time; to do this, I unpacked the save file and removed the "Oxygen Low" affliction.
If you're interested, I can send you this save.

Indeed, I didn’t pay attention to the fact that bot hands were cuffed behind. Then, of course, using a mask is impossible.
Thank you for you great work and good luck.
